Makefile

```diff
@@ -10,5 +10,5 @@ data:
 	go-bindata -o data/bindata.go -pkg data data/*.json
 
 run:
-	docker-compose build
-	docker-compose up --remove-orphans
+	sudo docker-compose build
```

Comment:
in what case did you need `sudo` here? docker-compose should be able to run without this